Sources/PackageGraph/PackageGraph.swift

```diff
@@ -43,8 +43,24 @@ public struct PackageGraph {
         self.rootPackages = rootPackages
         let inputPackages = rootPackages + rootDependencies
         self.packages = try! topologicalSort(inputPackages, successors: { $0.dependencies })
-        allTargets = Set(packages.flatMap({ $0.targets }))
-        allProducts = Set(packages.flatMap({ $0.products }))
+
+        allTargets = Set(packages.flatMap({ package -> [ResolvedTarget] in
```

Comment:
Is this a correct fix? I thought `allTarget` contains targets from all the packages.